net/ampr-ripd/Makefile

```diff
@@ -0,0 +1,98 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ampr-ripd
+PKG_VERSION:=2.4.1
+PKG_RELEASE:=1
+PKG_SOURCE_URL:=https://k2ie.net/ftp
```

Comment:
I dont think that we should use your website to download the latest version. It is different than the official site https://www.yo2loj.ro/hamprojects. We dont know what was changed, etc. This could be also security flaw. 

Why have you decided to use this approach? Why not download Git sources from your fork to see what have you done? Anyway, downloading from the official site should be the way.